src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/EmailPayloadAggregator.java

```diff
@@ -0,0 +1,60 @@
+package com.redhat.cloud.notifications.processors.email.aggregators;
+
+import com.redhat.cloud.notifications.models.EmailAggregation;
+import io.vertx.core.json.JsonObject;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+public abstract class EmailPayloadAggregator {
```

Comment:
Should we put 'Abstract' in the class name to make it more obvious when glancing at files?